### ISSUE TITLE
Initial installation - missing/relocated items 

### ISSUE THREAD

**Issue by Lou511** [sans-dfir/sift#68]
I recently installed the workstation on top of Ubuntu 14 LTS, and after the installation was completed (I used the bootstrap instal), I noticed the case, and mounting points folders were not present on the desktop.  I also noticed the cheat sheets were not present on the desktop.  I noticed the missing items, because approximately three months ago, I installed the workstation on the same Ubuntu build, and there were no missing items.  It would appear something changed with respect to the workstation installation process.  Ultimately, I did find the cases folder by changing directory to root, I did not find the cheat sheets, though.

I just want to make sure I installed correctly, since I am just starting to use SIFT.  Between this issue, and the other issue I posted - mounting a dd file and having to use a different syntax than the one listed in the cheat sheet, I figured I must've done something wrong with the install.

**Comment by ekristen** [comment]
There are a few different ways to run the bootstrap command ... how did you run it? If you did not run it with `-iys` then you will not get the PDFs, the background, folders etc ...

**Comment by Lou511** [comment]
I used the command combination associated with installing everything (theme and PDFs).  Interestingly, I ran the same installation steps a few months back, and nothing was missing.  However, I then ran into an issue involving mounting dd files (see my other post in the issue section).  As a result, I attempted to reinstall SIFT in case there was a bug with the original instal.  Would it help if I posted my instal log?

Thanks.

Luis

**Comment by ekristen** [comment]
@Lou511 have you had a chance to re-run the script?

**Comment by Lou511** [comment]
I did re run the script.  And the same thing occurred - couldn't execute the terminal commands I referenced in my original post.  Also, the folders and cheat sheets didn't download to the desktop either.  I did find the "cases" folder by cd /, and then doing an ls command.  Somehow, the folder made it into that directory instead of the desktop.

Out of curiosity, I loaded the VMware appliance file to see if I'd get the same issues.  The result - i have been able to mount images and I have the folders on the desktop.  I am not sure what the difference is between running the script and using the VMware file.  I thought I'd get identical installs, so to speak.  

Any suggestions or insight are welcomed.  Thank you for keeping your eye on my issue.

Thanks,

Luis

**Comment by ekristen** [comment]
You should get identical installs. There should be a sift-install.log available if you would like to gist that and post the link I'll take a look.

**Comment by Lou511** [comment]
Erik, please let me know how I can post a "link" to my install log.  I'm sorry I do not understand how to do so.  I can't paste the log into the comment box, since it's too long.  Please let me know what to do next to make the log available to you.

**Comment by ekristen** [comment]
paste it into https://gist.github.com and then paste the link here.

**Comment by Lou511** [comment]
I hope this works:

<script src="https://gist.github.com/Lou511/0695a3fb17ba5c62da70.js"></script>

Respectfully,

New guy

**Comment by Lou511** [comment]
Oops, maybe this will work.  Sorry.

https://gist.github.com/0695a3fb17ba5c62da70.git

**Comment by ekristen** [comment]
What is the exact command you are running? 

It also seems to be failing to pull the sift-files. 

Are you giving me the entire log file? It seems to just stop after sift-files.

**Comment by Lou511** [comment]
I doubled checked the install log.  I copied and pasted the entire log as is.  The script that I ran to install SIFT was the combination that gets everything (pdfs, theme, etc).  I only encountered this issue when I ran the script.  There are no problems running the VMware appliance file.  Just out of curiosity, I'm going to build another workstation using the same script I've tried in past installs.  Thanks again for all of your help.
